evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -374,7 +375,7 @@ def test_graph_partial_dependence_multiclass(logistic_regression_multiclass_pipe
     assert [fig_dict["data"][i]['name'] for i in range(3)] == ["class_0", "class_1", "class_2"]
 
     # Check that all the subplots axes have the same range
-    for suplot_1_axis, suplot_2_axis in [('axis2', 'axis3'), ('axis2', 'axis4'), ('axis3', 'axis4')]:
+    for suplot_1_axis, suplot_2_axis in [('axis', 'axis2'), ('axis', 'axis3'), ('axis2', 'axis3')]:
```

Comment:
We technically make four subplots but only fill in 3 of them. Since before we made a single `update_xaxes` call, all four suplots would get updated. Now, we only do it for the first three suplots (the ones that have data) so we should check `axes`, `axes2`, and `axes3`.